documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -5,11 +5,16 @@
 ### New features
 
 * Support for pnpm is now extended to 8.9.2.
+* Support for Nuget is now extended to 6.2 and Central Package Management in Nuget will now be supported for projects and solutions.
```

Comment:
Tweak suggestion: 
Nuget support extended to version 6.2 with Central Package Management now supported for projects and solutions.